roxctl/central/db/generate/generate.go

```diff
@@ -0,0 +1,172 @@
+package generate
+
+import (
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/stackrox/rox/pkg/buildinfo"
+	"github.com/stackrox/rox/pkg/errorhelpers"
+	"github.com/stackrox/rox/pkg/errox"
+	"github.com/stackrox/rox/pkg/images/defaults"
+	"github.com/stackrox/rox/pkg/mtls"
+	"github.com/stackrox/rox/pkg/renderer"
+	"github.com/stackrox/rox/pkg/roxctl"
+	"github.com/stackrox/rox/pkg/set"
+	"github.com/stackrox/rox/pkg/zip"
+	"github.com/stackrox/rox/roxctl/common"
+	"github.com/stackrox/rox/roxctl/common/environment"
+	"github.com/stackrox/rox/roxctl/common/flags"
+	"github.com/stackrox/rox/roxctl/common/logger"
+	"github.com/stackrox/rox/roxctl/common/zipdownload"
+)
+
+const (
+	centralDBCertGeneratePath = "/api/extensions/certgen/centraldb"
+)
+
+var (
+	errCreateCentralDBBundleSupported = errox.InvariantViolation.New("server does not support central db bundle functionality")
+	centralDBCertBundle               = set.NewFrozenStringSet(mtls.CACertFileName, mtls.CentralDBCertFileName, mtls.CentralDBKeyFileName)
+)
+
+type generateCommand struct {
+	// Properties that are bound to cobra flags.
+	config *renderer.Config
+
+	// Properties that are injected or constructed.
+	env environment.Environment
+
+	// timeout to make Central API call
+	timeout time.Duration
+}
+
+// Command represents the generate command.
+func Command(cliEnvironment environment.Environment) *cobra.Command {
```

Comment:
So, do I understand correctly that if you run `roxctl central db generate`, a bundle will be created _without_ any PVC? 

In that case, I think it makes sense to highlight this within the `Long` property of the cobra command, referring to the `pvc / hostpath` command to add persistency to the central DB.